src/NBitcoin/Networks/StratisMain.cs

```diff
@@ -60,6 +60,7 @@ public StratisMain()
             this.Consensus.ProofOfWorkReward = Money.Coins(4);
             this.Consensus.ProofOfStakeReward = Money.COIN;
             this.Consensus.MaxReorgLength = 500;
+            this.Consensus.MaxMoney = long.MaxValue;
```

Comment:
I'm not sure if we should max out here. 
For BTC we have 21 mil since it's the max supply that ca exist. 
Maybe for stratis we can just use a reasonably high value like 150 mil? 

Reason behind MaxMoney is to quickly check if the tx inputs are invalid so setting it to long.MaxValue basically will make this check redundant for stratis network.

Reply:
Other than that I'm fine with this PR.

Reply:
I agree that long.Max is big, but I just wanted tom move it without changing the functionality.

Reply:
Ok, in that case cool, let's merge it.